cypress/integration/locks.js

```diff
@@ -35,7 +35,7 @@ describe("Locks", function() {
     it("Should release locked A-EUR", function() {        
         getLock(8, 50).then(() => {
             cy.get("[data-testid=EthConfirmationReceivedPanel]").within(() => {
-                cy.contains("12. confirmation", { timeout: 20000 });
+                cy.contains("1. New lock", { timeout: 8000 });
```

Comment:
Do we need custom timeout at all, aren't we good with the default?

Reply:
indeed we should be good